data/assets/scene/solarsystem/missions/jwst/jwst.asset

```diff
@@ -12,18 +12,18 @@ local JWSTKernel = asset.syncedResource({
   Version = 1
 })
 
---[[local JWSTHorizons = asset.syncedResource({
+local JWSTHorizons = asset.syncedResource({
   Name = "JWST Horizons",
   Type = "HttpSynchronization",
   Identifier = "jwst_horizons",
   Version = 1
-})]]--
+})
 
 local model = asset.syncedResource({
   Name = "JWST Model",
   Type = "HttpSynchronization",
   Identifier = "jwst_model",
-  Version = 1
+  Version = 2
```

Comment:
This could probably be version 1, since it's the first version exposed to the users

Reply:
Since one version of the model has now been released, this comment is no longer valid :) Resolving